src/main/java/com/github/manolo8/darkbot/core/objects/slotbars/Item.java

```diff
@@ -80,10 +82,20 @@ public void update() {
         @Override
         public void update(long address) {
             this.address = address;
-            if (address == 0) return;
+            if (address == 0) {
+                resetTimer();
+                return;
+            }
 
             this.startTime = API.readMemoryDouble(address + 80);
             this.itemDelay = API.readMemoryDouble(address + 88);
         }
+
+        public void resetTimer() {
```

Comment:
A function inside itemtimer does not need to call itself resetTimer, this is already the timer
just rename it to reset
```suggestion
        public void reset() {
```